src/main/java/DoneCommand.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Represents a command by the user to mark a task in the list of tasks as done.
+ * An DoneCommand object has a task number to represent the task to be marked done.
+ */
+public class DoneCommand extends Command {
+    private int taskNum;
+
+    /**
+     * Instantiates a done command.
+     *
+     * @param taskNum The task number to be marked as done.
+     */
+    public DoneCommand(int taskNum) {
+        this.taskNum = taskNum;
+    }
+
+    /**
+     * Executes the command by marking the specified task as done, replying the user,
+     * and updating the list of tasks stored in the computer.
+     *
+     * @param tasks List of all the tasks of the user.
+     * @param ui Replies the user.
```

Comment:
Maybe you could elaborate a little more on the ui parameter? It appears to be abit unclear about how the ui will reply the user.